docs/manual/scala/releases/Migration14.md

```diff
@@ -13,12 +13,21 @@ The version of Lagom can be updated by editing the `project/plugins.sbt` file, a
 addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0")
 ```
 
-Lagom 1.4.0 also requires Sbt 0.13.16 or later. If your existing project is using a previous version of Sbt, you will need to upgrade it by editing the `project/build.properties` file. For example:
+Lagom 1.4.0 also requires Sbt 0.13.16 or later (recommended sbt 1.x). If your existing project is using a previous version of Sbt, you will need to upgrade it by editing the `project/build.properties` file. For example:
 
 ```
 sbt.version=0.13.16
 ```
 
+We recommend you migrate your projects to sbt 1.x by editing the `project/build.properties` file. For example:
+
+```
+sbt.version=1.1.4
```

Comment:
We may simplify here by driving people to upgrade to sbt 1.1.4 directly instead. 
So, instead of saying: upgrade to 0.13.16, but we actually recommend upgrading to 1.1.4. 

We can say: 
> Lagom 1.4.0 also requires Sbt 0.13.16 or later (recommended sbt 1.x). If your existing project is using a previous version of Sbt, you will need to upgrade it by editing the `project/build.properties` file. For example:
> ```
> sbt.version=1.1.4
> ```